lib/cfme/cloud_services/inventory_upload.rb

```diff
@@ -0,0 +1,56 @@
+module Cfme
+  module CloudServices
+    class InventoryUpload
+      def self.upload(targets)
```

Comment:
On the API side we were going with "sync".  What are your thoughts on terminolgy here.

Reply:
Yeah we could call it `InventorySync.sync` or something like that, I'm not really tied to a name

Reply:
Done, let me know if you or @abellotti like a different name better

Reply:
sync_queue sounds about right, returning the task id for the API sync action. 👍